backend/src/codegen.yml

```diff
@@ -0,0 +1,13 @@
+overwrite: true
+schema: 'http://localhost:3000/graphql'
+documents: 'src/**/*.graphql'
+generates:
+  src/graphql/generated.tsx:
+    plugins:
+      - 'typescript'
+      - 'typescript-operations'
+      - 'typescript-react-apollo'
+    config:
+      withHooks: true
+      withHOC: false
+      withComponent: false
```

Comment:
You don't need this here — it's codegen file for react, you don't need it in backend

Reply:
Done!